src/modules/form/components/FormNavigationContainer.tsx

```diff
@@ -26,7 +26,7 @@ const FormNavigationContainer = ({
   children,
 }: FormNavigationProps) => {
   const isTiny = useIsMobile('sm');
-  const focusTargetId = 'focusable-form';
+  const focusTargetId = 'focusable-form'; // This expects that only 1 form is rendered at a time.
```

Comment:
only 1 form with _form navigation_ enabled, right? On the Enrollment Details we have lots of dynamic-view forms rendered alongside each other, but that doesn't seem like it would cause an issue here.

Reply:
ok. thanks!